localstack/services/awslambda/invocation/runtime_executor.py

```diff
@@ -100,13 +99,22 @@ def prepare_image(target_path: Path, function_version: FunctionVersion) -> None:
             image_name=get_image_name_for_function(function_version),
         )
     except Exception as e:
-        LOG.error("Exception: %s", e)
+        if LOG.isEnabledFor(logging.DEBUG):
+            LOG.exception(
+                "Error while building prebuilt lambda image for '%s'",
+                function_version.qualified_arn,
+            )
+        else:
+            LOG.error(
+                "Error while building prebuilt lambda image for '%s', Error: %s",
+                function_version.qualified_arn,
+                e,
+            )
```

Comment:
might be a nit but `LOG.error(..., exc_info=LOG.isEnabledFor(logging.DEBUG))` should do basically the same, :thinking:

Reply:
Yes, but the exception message differs as well. In the end it might not matter, will leave it for now.